nixos/modules/services/networking/dhcpcd.nix

```
@@ -146,6 +156,21 @@ in
       '';
     };
 
+    networking.dhcpcd.wait = mkOption {
+      type = types.enum [ "background" "any" "IPv4" "IPv6" "both" "if-carrier-up" ];
```

```suggestion
      type = types.enum [ "background" "any" "ipv4" "ipv6" "both" "if-carrier-up" ];
```

Small nitpick: We're already using lowercase `ipv4` and `ipv6` (lowercase) in the module system so I think we should also use the lowercase spelling here for consistency.